src/Uno.UWP/UI/Core/CoreWindow.cs

```diff
@@ -31,8 +31,10 @@ public Point PointerPosition
 			set => _pointerPosition = value;
 		}
 
+#if !__WASM__
```

Comment:
Would be great to add [support for Android](https://developer.android.com/reference/android/view/PointerIcon.html) (And also iOS since mouse are supported in iOS 13 ... but I'm pretty sure that we cannot customize the cursor :/)